codice/ddf
# DDF-2382 Update the registry to use metacard attribute taxonomy

#### What does this PR do?
The registry apps RegistryObjectMetacardType was updated to conform to the new metacard attribute taxonomy.
This means 
1) RegistryObjectMetacardType now only has registry specific attributes. For 'common' attributes we now use the names from the new attribute classes.
2) All registry specific attributes have been renamed to be `registry.<current-attribute-name>`
Exception to this is the transient attributes which is `registry.local.<current-attribute-name>`
3) The registry-api blueprint was built-up and now exposes the registry metatype as a service. 
4) The registry transformer/package converter no longer parses out the content type from the metacard and the content type is no longer used as a filter for evaluating registry metacards 

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@clockard @gordocanchola @vinamartin @brianfelix @jlcsmith @brendan-hofmann 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@figliold
@jlcsmith 

#### How should this be tested?
Install DDF, Start the Registry App and confirm the identity node is present in the local nodes tab. Confirm edits to the nodes are retained and that connections can still be made using the remote nodes tab.

#### Any background context you want to provide?
This ticket will need to be backported to the 2.9.x branch.

#### What are the relevant tickets?
DDF-2382 

#### Screenshots (if appropriate)
#### Checklist:
